Document support for OAuth apps (#651)
gristlabs/grist-help: help/en/docs/rest-api.md

## Authentication

To access the Grist API, you'll need an API key.  An API key is owned by a single
user, and has the same permissions as that user.

!!! note "Connected apps"
    An API key carries your full account access. For tools that act on your data on your behalf --
    partner products, internal apps, AI agents -- a [connected app](connected-apps.md) is a safer
    choice: it can be authorized for specific documents and revoked individually, for greater
    security, control, and visibility.
    For building such tools, see [OAuth apps](oauth-apps.md).

### API keys

To enable API access for yourself, visit the [Developer page](https://docs.getgrist.com/account/developer)
of your account settings. You can always find this page by clicking your profile picture or

> initial on the top right of the screen, and selecting "Account settings":

Reviewer: ```suggestion
initial on the top right of the screen, and selecting 'Account settings':
```

*![Account settings in user menu](images/api/user-menu-account-settings.png)*
{: .screenshot-half}

The 'Developer' page has a section for API keys:

![API key settings](images/api/api-key-settings.png)

> Click on the "Create" button to create an API Key. Copy it for use when making API calls.

Reviewer: ```suggestion
Click on the 'Create' button to create an API Key. Copy it for use when making API calls.
```

> You can revoke your API key by clicking the "Remove" button at any time.

Reviewer: ```suggestion
You can revoke your API key by clicking the 'Remove' button at any time.
```

You'll then have the option to create a new one if you wish.

To test your api key, try this from the command-line (substituting

> your api key):

Reviewer: ```suggestion
To test your API key, try this from the command-line (substituting
your API key):
```
